edge/pkg/eventbus/eventbus.go

```diff
@@ -106,9 +109,11 @@ func (eb *eventbus) pubCloudMsgToEdge() {
 		switch operation {
 		case messagepkg.OperationSubscribe:
 			eb.subscribe(resource)
```

Comment:
`subscribe` does not raise err when subscribe failed, the time to insert to db need more concern.

Reply:
Thanks, I had modified.